examples/tests/igraph_to_directed.c

```diff
@@ -0,0 +1,43 @@
+
+#include <igraph.h>
+
+#include "test_utilities.inc"
+
+int main() {
+    igraph_t ug;
+    igraph_t dg;
+
+    igraph_small(&ug, 6, /* directed= */ 0,
+                 2,0, 1,4, 3,2, 2,3, 0,4, 5,0, 2,3, 5,3, 2,5, 0,1,
```

Comment:
The duplicate edges are explicitly in here to test the case with parallel edges?

Reply:
Yes.